src/SourcemapToolkit.SourcemapParser/Properties/AssemblyInfo.cs

```diff
@@ -35,4 +35,6 @@
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
 
+// Sees internals to share extension methods
+[assembly: InternalsVisibleTo("SourcemapToolkit.CallstackDeminifier, PublicKey = 0024000004800000940000000602000000240000525341310004000001000100fb6b178a31a5d56c14ad5de5e8f471e4b37ef4291eb74544d7477a42ba0769e04230fada2df8722623a26b018ca2aeb66b01e5a456dc56194f0e9826837151906ca02f3158111e41c3dd770ca307825402ff75d9602929b5594fe9a91aff9093d19cb2b35313452b48ab8ef3b7a997a09ee5f8c7c5285c327a30e3f16ef422a4")]
```

Comment:
This doesn't seem healthy. Using `InternalsVisibleTo` for unit tests (the line below) might be okay, but if product code needs to access an internal then we should make that internal public.

Reply:
I'm not sure I agree with that.  These two assemblies are pretty tightly linked, and I actually don't like that so much of their internal logic is being exposed to the outside world. I'd prefer if they only exposed the APIs a consumer is supposed to use.

If you feel strongly I can change it though, let me know.

Reply:
I do not feel strongly about it.